doc/bitcoin-conf.md

```diff
@@ -61,4 +61,9 @@ Windows | `%APPDATA%\Bitcoin\` | `C:\Users\username\AppData\Roaming\Bitcoin\bitc
 Linux | `$HOME/.bitcoin/` | `/home/username/.bitcoin/bitcoin.conf`
 macOS | `$HOME/Library/Application Support/Bitcoin/` | `/Users/username/Library/Application Support/Bitcoin/bitcoin.conf`
 
-You can find an example bitcoin.conf file in [share/examples/bitcoin.conf](../share/examples/bitcoin.conf).
+An example configuration file is generated by [contrib/generate-example-bitcoin-conf.sh](../contrib/generate-example-bitcoin-conf.sh) during `make install`. The output is placed in [share/examples/bitcoin.conf](../share/examples/bitcoin.conf). If you don't see this file, you can generate it by running `./contrib/generate-example-bitcoin-conf.sh` from the root directory, or you can run `make install` again. To use, copy the example file into the correct data directory for your system and edit it there, like so:
```

Comment:
You can't include the link to the example configuration file here because we're removing it from the repo under this PR. This link is invalid link if left on the repo. It is only viable locally, after the user has generated the example file.

suggestions here:
``` diff
diff --git a/doc/bitcoin-conf.md b/doc/bitcoin-conf.md
index 98517f2fc..3babf176a 100644
--- a/doc/bitcoin-conf.md
+++ b/doc/bitcoin-conf.md
@@ -61,7 +61,10 @@ Windows | `%APPDATA%\Bitcoin\` | `C:\Users\username\AppData\Roaming\Bitcoin\bitc
 Linux | `$HOME/.bitcoin/` | `/home/username/.bitcoin/bitcoin.conf`
 macOS | `$HOME/Library/Application Support/Bitcoin/` | `/Users/username/Library/Application Support/Bitcoin/bitcoin.conf`
 
-An example configuration file is generated by [contrib/generate-example-bitcoin-conf.sh](../contrib/generate-example-bitcoin-conf.sh) during `make install`. The output is placed in [share/examples/bitcoin.conf](../share/examples/bitcoin.conf). If you don't see this file, you can generate it by running `./contrib/generate-example-bitcoin-conf.sh` from the root directory, or you can run `make install` again. To use, copy the example file into the correct data directory for your system and edit it there, like so:
+An example configuration file can be generated by [contrib/generate-example-bitcoin-conf.sh](../contrib/generate-example-bitcoin-conf.sh).
+You can run this script directly or have it automatically executed during the `make install` stage.
+The output is placed under `share/examples/bitcoin.conf`.
+To use the generated configuration file, copy the example file into your data directory and edit it there, like so:
```

Reply:
ah, good catch!